libs/features/georoutingv2/georoutingv2.js

```diff
@@ -236,26 +236,30 @@ function buildContent(currentPage, locale, geoData, locales) {
 
 async function getDetails(currentPage, localeMatches, geoData) {
   const availableLocales = await getAvailableLocales(localeMatches);
-  if (availableLocales.length > 0) {
-    const georoutingWrapper = createTag('div', { class: 'georouting-wrapper fragment' });
-    currentPage.url = window.location.hash ? document.location.href : '#';
-    if (availableLocales.length === 1) {
-      const content = buildContent(currentPage, availableLocales[0], geoData);
-      georoutingWrapper.appendChild(content);
-      return georoutingWrapper;
-    }
-    const sortedLocales = availableLocales.sort((a, b) => a.languageOrder - b.languageOrder);
-    const tabsContainer = createTabsContainer(sortedLocales.map((l) => l.language));
-    georoutingWrapper.appendChild(tabsContainer);
-
-    sortedLocales.forEach((locale) => {
-      const content = buildContent(currentPage, locale, geoData, sortedLocales);
-      const tab = createTab(content, locale.language);
-      georoutingWrapper.appendChild(tab);
-    });
+  if (!availableLocales.length) return null;
+  const { innerWidth } = window;
```

Comment:
`innerWidth` might not always take scrollbars into consideration, `matchMedia` could be a good alternative. You'd have to properly resize the window for unit testing, but it might be more consistent.

Reply:
Good to know! Since `innerWidth` is several times faster than `matchMedia`, I think I'll leave it as is.